src/lib/learn-client/types.ts

```diff
@@ -118,10 +118,65 @@ export type AllCollectionsProductOptions = {
   sidebarSort?: boolean
 }
 
+/**
+ * PRODUCT LANDING PAGES -------------------------------
+ */
+
+export interface ProductPageBlockBrandedCallout {
```

Comment:
💭 I wonder if the `ProductPage` prefix needs to be even _more_ explicit, since its tutorials specific. I'm not sure its necessary but I _could_ see room for confusion on what product page is referring to.

Reply:
Ooh great point 🤔

For now I took another look at this, and realized I really like the pattern of `import { Thing as ClientThing }` I've seen in other areas of the app where we import `learn-client/types` stuff into front-end page & component files. So I'm doing eg `import { ProductPageBlock as LearnClientProductPageBlock } from 'lib/learn-client/types'`. A bit wordy but hopefully feels more consistent with other imports ✨ 

Definitely open to iterating on the naming here further - will keep it in mind as I work through individual components; I think if I end up extending & otherwise re-using the `ProductPageBlock<BlockType>` types, it'll probably come up again ♻️